pyproject.toml

```diff
@@ -0,0 +1,73 @@
+[tool.poetry]
+name = "mxcube3"
+version = "1.0.0"
+license = "LGPL-3.0-or-later"
+description = "MXCuBE Web user interface"
+authors = ["The MXCuBE collaboration <mxcube@esrf.fr>"]
+maintainers = [
+    "MXCuBE collaboration <mxcube@esrf.fr>",
+]
+readme = "README.md"
+homepage = "http://github.com/mxcube/mxcubeweb"
+repository = "http://github.com/mxcube/mxcubeweb"
+documentation = ""
+keywords = ["mxcube", "mxcube3", "mxcubeweb"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: End Users/Desktop",
+    "Natural Language :: English",
+    "Topic :: Scientific/Engineering",
+]
+
+[tool.poetry.dependencies]
+python = ">=3.8,<3.11"
```

Comment:
Are we dropping Python3.7 support?

Reply:
Not really, we can keep 3.7 if its needed it is indeed supported by mxcuebcore. I just thought it was time to move a head and did not really see the reason for keeping 3.7 thats all

Reply:
@marcus-oscarsson I'm happy to drop it a few months early, but we should probably update the Python version constraints for MXCuBE Core to be consistent.

Reply:
Yes, that's true, Ill that :)